recipes/prospector/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "prospector" %}
+{% set version = "0.12.2" %}
+{% set sha256 = "eee570da89ac0fec8d746fc72c97e1a8aca9dd82161cac175d78ddd0d5170487" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  entry_points:
+    - prospector = prospector.run:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - pylint >=1.5.6
+    - pylint-celery >=0.3
+    - pylint-django >=0.7.2
+    - pylint-flask >=0.3
+    - pylint-plugin-utils >=0.2.3
+    - pylint-common >=0.2.2
+    - requirements-detector >=0.4.1
+    - setoptconf >=0.2.0
+    - dodgy >=0.1.9
+    - pyyaml
+    - mccabe >=0.5.0
+    - pyflakes >=0.8.1
+    - pycodestyle >=2.0.0
+    - pep8-naming >=0.3.3
+    - pydocstyle >=0.1
+    # Optional dependencies.
+    - frosted >=1.4.1
+    - vulture >=0.6
+    - pyroma >=2.0.2
+
+test:
+  imports:
+    - prospector
+    - prospector.config
+    - prospector.formatters
+    - prospector.profiles
+    - prospector.tools
+    - prospector.tools.dodgy
+    - prospector.tools.frosted
+    - prospector.tools.mccabe
+    - prospector.tools.pep257
+    - prospector.tools.pep8
+    - prospector.tools.profile_validator
+    - prospector.tools.pyflakes
+    - prospector.tools.pylint
+    - prospector.tools.pyroma
+    - prospector.tools.vulture
+
+  commands:
+    - prospector --help
+
+about:
+  home: https://github.com/landscapeio/prospector
+  license: GPL 2
```

Comment:
Proposing to include the license file in PR ( https://github.com/landscapeio/prospector/pull/184 ).